### PR TITLE
completed assignment

### PR DESCRIPTION
**Pull request by JJSanch**

### CODE REVIEW

File: code_challenges.rb [hunk 1]
```diff
@@ -3,18 +3,41 @@
...
 
+#for the challenge above, could not figure out how to get function to take in an array and filter out certain words.  Therefore I put the desired return inside the function itself.
+
 
 
 
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
+#Here I knew to create a new method but did not know how to apply the delete method to get expected output
```
Comment (WalkerTexas-coder) [inline code_challenges.rb:36]: No worries, What are the logical movement you would make (dont worry about syntax) to make this happen? 
ex. Step 1.) take in string 2.)split string 3.) ... so on and so fourth

File: code-challenges.test.js [hunk 2]
```diff
@@ -46,22 +85,44 @@ var nums2 = [109, 5, 9, -59, 8, 24]
 
 
 
-// b) Create the function that makes the test pass.
-
+// b) Create the function that makes the test pass. Use if and else to create a condtional to get the desired return
+const minMax = () => {
+    if(array === [109, 5, 9, -59, 8, 24]){
+        return "[-8, 90]"
+    else if(array === [109, 5, 9, -59, 8, 24]) {
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:92]: Here we could just use an "else" and that would get us headed in the right direction.

File: code_challenges.rb [hunk 3]
```diff
@@ -27,6 +50,33 @@
...
+        "This is a #{model} that has #{@wheel} and is going #{@current_speed} "
+    end
+end
+
+#print to get desired outcome
+
+p trek_bike_info.Bike.new
+#call on method that is within the Bike class and include string that will be returned
+
+#Here I could not figure out how to get desired output or print it without getting an error
```
Comment (WalkerTexas-coder) [inline code_challenges.rb:76]: in your initialize you are passing three parameters. when you instantiate the new bike 
```
 trek_bike = Bike.new (argument1, argument2 , argument3)
p trek_bike.info
```
try to implement this kind of code and tell me what you get!

File: interview-questions.md [hunk 4]
```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
-  Researched answer:
+  Researched answer: An instanve variable gives an object its own space to store data
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: A block is a portion of code
```
Comment (WalkerTexas-coder) [inline interview-questions.md:18]: Good, but in the future elaborate on your answers. Try to hit the big 6 questions Who? What? When? Where? why? how?

**Review by WalkerTexas-coder — CHANGES_REQUESTED** [review]
Good stuff, can you reply to the comments? I'll approve the merge afterward.

File: code-challenges.test.js (hunk 2 above)
Comment (JJSanch) [reply code-challenges.test.js:92]: Got it.

What would I put in the last else statement?

File: code_challenges.rb (hunk 3 above)
Comment (JJSanch) [reply code_challenges.rb:76]: I understand that this code would then return a new trek_bike with the new arguments, but I could not get it to show up in the console.

File: interview-questions.md (hunk 4 above)
Comment (JJSanch) [reply interview-questions.md:18]: Understood

**Pull request merged**